tests/phpunit/class-wc-stripe-payment-gateway-test.php

```
@@ -1031,4 +1031,130 @@ public function provide_test_disable_subscription_edit_for_india(): array {
+		$filter = function () use ( $mock_response ) {
+			return $mock_response;
+		};
+		add_filter( 'pre_http_request', $filter );
+
+		$this->gateway->update_fees( $order, 'txn_invalid', true );
+
+		remove_filter( 'pre_http_request', $filter );
+
+		$this->assertEquals( 1.50, (float) $order_helper->get_stripe_fee( $order ) );
+		$this->assertEquals( 48.50, (float) $order_helper->get_stripe_net( $order ) );
+	}
```

PHPMD (2.15.0)

[error] The method test_update_fees_with_api_error_leaves_meta_unchanged is not named in camelCase. (undefined)

(CamelCaseMethodName)